src/app/poll/mobile-poll-table.component.scss

```diff
@@ -27,11 +27,10 @@
 }
 
 .poll-table td.suggested-date {
-  @media (min-width: map-get($grid-breakpoints, sm)) {
-    width: 150px;
-  }
-  @media (min-width: map-get($grid-breakpoints, md)) {
-    width: 180px;
+  width: 180px;
+
+  @media (max-width: 430px) {
```

Comment:
Wieso wird hier kein grid Breakpoint mehr genutzt?

Reply:
Weil sich das Layout nur an dieser Stelle bricht und die Breite unterhalb der kleinsten Größe liegt. Bei dem Problem hatte ich auch nicht das Gefühl, dass man das sinnvoll in das bestehende Grid einpflegen kann oder es sich lohnt die Breite dort aufzunehmen.